--- origsrc/elmdentica/src/Makefile.am	2012-09-13 01:09:42.000000000 -0500
+++ src/elmdentica/src/Makefile.am	2012-09-13 04:10:42.434221000 -0500
@@ -25,7 +25,7 @@ BUILT_SOURCES = statusnet_Common.h statu
 CLEANFILES = $(BUILT_SOURCES) .sources-ts
 $(BUILT_SOURCES): .sources-ts
 .sources-ts: statusnet.azy
-	azy_parser -eHn -m common-impl,common-headers -o $(top_builddir)/src $(top_builddir)/src/statusnet.azy
+	azy_parser -eHn -m common-impl,common-headers -o $(top_builddir)/src $(top_srcdir)/src/statusnet.azy
 	touch $(top_builddir)/src/.sources-ts
 
 bin_PROGRAMS = elmdentica
